weblate/templates/screenshots/screenshot_list.html

```diff
@@ -22,7 +30,13 @@
 <div class="panel panel-default">
 <div class="panel-heading"><h4 class="panel-title">{% trans "Add new screenshot" %}</h4></div>
   <div class="panel-body">
-  {{ add_form|crispy }}
+  <p id="paste-screenshot-info-label" class="text-center"></p>
+  <button id="paste-screenshot-btn" class="btn" type="button">
+    {% trans "Paste from clipboard" %}
+  </button>
```

Comment:
Can this be placed in the form, just next to the screenshot field:

![image](https://github.com/user-attachments/assets/010525ce-009f-4a28-90d3-982daac47d77)

It would need introducing rendering helper here and adding `Div` element to render the additional content:

https://github.com/WeblateOrg/weblate/blob/c006da8c8954d56fcac444833407e42e12f014cc/weblate/screenshots/forms.py#L25-L52

An example can be found here:

https://github.com/WeblateOrg/weblate/blob/c006da8c8954d56fcac444833407e42e12f014cc/weblate/trans/forms.py#L2305-L2312

Reply:
Okay. Thanks for the examples. I found where the form was created but I thought it would be better if I didn't change it here but change the markup instead. However I think it's better if it's closer to the input.

Reply:
Part of the button stayed, removed in d944d52623.

Reply:
😳 Thanks for the fix.